trpcultivate_phenotypes/src/TripalCultivateValidator/TripalCultivatePhenotypesValidatorBase.php

```diff
@@ -201,4 +212,39 @@ public function getConfigAllowNew() {
 
     return $allownew;
   }
+
+  /**
+   * Split or explode a data file line/row values into an array using a delimiter.
+   * 
+   * @param string $row
+   *   A line in the data file.
+   * 
+   * @return array
+   *   An array containing the values extracted from the line after splitting it based
+   *   on a delimiter value.
+   */
+  public function splitRowIntoColumns(string $row) {
+    // Delimiter:
+    $delimiter = $this->context['delimiter'] ?? ''; 
+    if (empty($delimiter)) {
+      throw new \Exception(t('No delimiter provided.'));
```

Comment:
We don't translate exception messages as they are not meant for the UI but instead for the developer.